### PR TITLE
Twig 2.0

### PR DESCRIPTION
**Pull request by GGKapanadze**

### CODE REVIEW

File: .changeset/empty-dogs-invent.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@ilo-org/maestro": minor
+---
+
+story creation
```
Comment (bashlk) [inline .changeset/empty-dogs-invent.md:5]: Should probably say initial version of the package right?

**Comment by bashlk** [comment]
Fixed the build command in netlify settings for  ilo-ui-twig-develop. I didn't change it for the others because it would break the build until this branch is merged.

@justintemps let us know if we should update the build command for those sites as well or you can do it. (Just need to remove the non existing react:install and twig:install commands from the build command)

**Comment by justintemps** [comment]
Thanks @bashlk great if you could do it.

**Comment by bashlk** [comment]
@justintemps Corrected the build command for all the storybook sites. Build might now fail until the new changes are merged to main.

**Pull request merged**